metadata-ingestion/src/datahub/ingestion/source/qlik_sense/qlik_sense.py

```diff
@@ -252,17 +257,23 @@ def _gen_charts_workunit(self, charts: List[Chart]) -> Iterable[MetadataWorkUnit
 
             yield self._gen_entity_status_aspect(chart_urn)
 
+            custom_properties = {
+                "Dimension": str(chart.qDimension),
+                "Measure": str(chart.qMeasure),
+            }
+
             yield MetadataChangeProposalWrapper(
                 entityUrn=chart_urn,
                 aspect=ChartInfoClass(
-                    title=chart.qId,
-                    description=chart.qType,
+                    title=chart.title,
```

Comment:
the charts also need a container aspect

<img width="983" alt="image" src="https://github.com/datahub-project/datahub/assets/12566801/b140b583-2e07-4338-be81-92aa2c8dad53">